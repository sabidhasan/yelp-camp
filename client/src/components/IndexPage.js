import React from 'react'

import Banner from './Banner'
import LandingText from './LandingText'
import LandingTiles from './LandingTiles'
import DiscoverTile from './DiscoverTile'
import Campgrounds from './Campgrounds'

const IndexPage = (props) => {
  return (
    <React.Fragment>
      <Banner {...props} />
      <LandingText {...props} />
      <LandingTiles {...props} />
      <DiscoverTile {...props}/>
      <Campgrounds {...props} />
    </React.Fragment>
  )
};

export default IndexPage
