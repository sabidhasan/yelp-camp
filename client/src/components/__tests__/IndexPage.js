import React from 'react'
import toJson from 'enzyme-to-json'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import IndexPage from '../IndexPage'

configure({ adapter: new Adapter() });

describe('<IndexPage />', () => {
  let component;
  let props = {};

  beforeEach(() => {
    component = shallow(<IndexPage { ...props } />)
  })

  it('IndexPage should be defined', () => {
    expect(component).toBeDefined();
  });

  it('IndexPage component snapshot matches what is expected', () => {
    expect(toJson(component)).toMatchSnapshot();
  })
})
