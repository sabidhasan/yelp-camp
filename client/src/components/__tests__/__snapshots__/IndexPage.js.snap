// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexPage /> IndexPage component snapshot matches what is expected 1`] = `
<Fragment>
  <Banner />
  <LandingText />
  <LandingTiles />
  <DiscoverTile />
  <Campgrounds />
</Fragment>
`;
